lib/todolist_web/controllers/task_controller.ex

```diff
@@ -0,0 +1,9 @@
+defmodule TodolistWeb.TaskController do
+  use TodolistWeb, :controller
+
+  def index(conn, _params) do
+    tasks = Todolist.Tasks.Task.get_all() |> Enum.map(fn(%Todolist.Tasks.Task{name: name}) -> name end)
```

Comment:
Not sure if you ran `mix format`

Reply:
Also, the `map` function could be cleaner:

```elixir
Enum.map(fn task -> task.name end)
```

or even

```elixir
Enum.map(& &1.name)
```

Reply:
Done.